packages/db/src/meta/interface.ts

```diff
@@ -34,15 +33,9 @@ export const forAttachAndSchema = <C extends Collections>(options: {
 }) => {
   const { attach, schema } = options;
 
-  const connect = (
-    config: TruffleConfig | ConnectOptions<C> | undefined
-  ): Db<C> => {
-    // TODO: get rid of @truffle/config stuff and rename field to "directory"
+  const connect = (config: ConnectOptions<C> | undefined): Db<C> => {
```

Comment:
I believe everywhere else uses `?` instead of `| undefined`:

```suggestion
  const connect = (config?: ConnectOptions<C>): Db<C> => {
```

Reply:
Oh oops I guess technically this should've gone on the other PR. Well whatever, pick whichever way you want the merge conflict ;)